Documentation to support GCP for Chef Automate HA (#8214)
chef/automate: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md

We recommend using Chef Infra Server managed by Automate HA to have high availability for both Automate and Infra Server. External Standalone Infra Server will violate this high availability requirement.

## Provisioning

> To Deploy On-Premises Infra we need to provision the resources which are required, Provisioning can be done using following types of infrastructures:
> 
> **Existing infrastructure**
> 
> **Existing Cloud infrastructure**
> - AWS
> - Google Cloud Platform
> 

Reviewer: ```suggestion
To Deploy On-Premises Infra user needs to provision the resources manually that are required. Provisioning can be done on the following types of infrastructures:

- Existing infrastructure
- Existing Cloud infrastructure
    - AWS
     - Google Cloud Platform

```

Author: done

## Backup and Restore

> In On-premises deployment of Automate HA, we support [**Network File System (NFS)**](/automate/ha_backup_restore_file_system/) or [**Object Storage (S3/MinIO/Google-Cloud-Storage)**](/automate/ha_backup_restore_object_storage/) for taking backup.

Reviewer: ```suggestion
In On-premises deployment of Automate HA, we support [**Network File System (NFS)**](/automate/ha_backup_restore_file_system/) or [**Object Storage (S3/MinIO/Google Cloud Storage)**](/automate/ha_backup_restore_object_storage/) for taking backup.
```

Author: done
